lib/agent0/agent0/hyperdrive/exec/execute_agent_trades.py

```diff
@@ -123,6 +125,20 @@ async def async_execute_single_agent_trade(
             )
         trade_results.append(trade_result)
 
+    # TODO to avoid adding a post action in base policy, we only call post action
```

Comment:
I'm not following this TODO. We have a base policy that HyperdriveBasePolicy inherits from.

Reply:
I went down the rabbit hole of adding this to the base agent and policy, and ultimately got to a dead end due to generics. Base policy isn't strictly a hyperdrive base policy, because hyperdrive base policy is explicitly setting the generic types of the market, whereas base policy takes a generic.